### ISSUE TITLE
layman.add does not work with unofficial overlays

### ISSUE THREAD

**Issue by godlike64** [saltstack/salt#25949]
A couple of months ago (coupled with the Gentoo Portage rework), layman started having a division between official and unofficial overlays.

When an overlay is unofficial, the user is prompted about this and asked to either accept adding it or deny it.

It seems this has broken Salt's layman module with regards to adding an overlay. I tried it and got no output (no error or anything) on the master, and when I went to look at the minion's log, I saw:
```
2015-08-02 00:28:10,513 [salt.loaded.int.module.cmdmod][ERROR   ][6219] Command 'layman --quietness=0 --add bitcoin' failed with return code: 1
2015-08-02 00:28:10,513 [salt.loaded.int.module.cmdmod][ERROR   ][6219] output: 
 * Adding overlay...
 * Overlay "bitcoin" is not an official. Continue install? [y/n]: 
 * CLI: Errors occurred processing action add
 * Exception caught installing repository 'bitcoin' : EOF when reading a line
```

**Comment by jfindlay** [comment]
@godlike64, thanks for the report.  This patch should fix the problem.

**Comment by godlike64** [comment]
@jfindlay thanks for the commit, I don't have currently with me a master sandbox to pull and test it, but I gather the stdin=y shouldn't cause problems with official overlays (when the question is not asked)?

**Comment by jfindlay** [comment]
@godlike64, from my observations in testing it shouldn't cause a problem.  If it does, we can revisit this issue.  Thanks.

**Comment by cachedout** [comment]
Closed via #25970
